charts/linkerd2/values.yaml

```diff
@@ -132,6 +132,47 @@ heartbeatSchedule: "0 0 * * *"
 # prometheus configuration
 prometheusImage: prom/prometheus:v2.15.2
 prometheusLogLevel: *controller_log_level
+prometheusExtraArgs: {}
+  # log.format: json
+prometheusAlertmanagers: []
+  # - scheme: http
+  #   static_configs:
+  #   - targets:
+  #     - "alertmanager.linkerd.svc:9093"
+prometheusExtraConfigMapMounts: []
+  # - name: alerting-rules
+  #   mountPath: /etc/prometheus
+  #   subPath: "alerting_rules.yml"
+  #   configMap: linkerd-prometheus-rules
+  #   readOnly: true
```

Comment:
As per my comment above we can get rid here of `mountPath` and `readOnly`

Reply:
Done!